package.json

```diff
@@ -43,7 +43,7 @@
     "@types/type-is": "^1.6.2",
     "chance": "^1.0.18",
     "gavel": "^6.1.3",
-    "globby": "^10.0.1",
+    "globby": "^9.2.0",
```

Comment:
This was not an intended update — breaking changes not really, not right now. I need to read the changeling and then I'll schedule the upgrade.